File: Laboratorio-01/ejercicio1.py
def sumarArreglo (arreglo):
    suma = 0;
    
    for i in range(0, len(arreglo)):
        suma += arreglo[i]
    print(f"La suma total del arreglo es: {suma}")

arreglo = [1,2,3,4,100];
sumarArreglo(arreglo)

File: Laboratorio-01/ejercicio1.java
public class SumaArreglos {

	public static void main(String[] args) {
		int [] arreglo = {1, 2, 3, 4, 100};
		sumarArreglo(arreglo);
	}
	
	public static void sumarArreglo (int[] arreglo) {
		int suma = 0;
		for (int i=0; i < arreglo.length; i++) {
			suma += arreglo[i];
		}
		System.out.println("La suma total del arreglo es: " + suma);
	}

}
